app/templates/events/view/index.hbs

```diff
@@ -2,15 +2,17 @@
   <div class="row">
     <div class="eight wide column">
       <Events::View::Overview::EventSetupChecklist @data={{this.model}} />
-      <Events::View::Overview::EventSponsors
-        @data={{this.model.sponsors}}
-        @columns={{this.columns}}
-        @page={{this.page}}
-        @per_page={{this.per_page}}
-        @search={{this.search}}
-        @sort_by={{this.sort_by}}
-        @sort_dir={{this.sort_dir}}
-        @filterOptions={{this.filterOptions}} />
+      {{#if this.showSponsors}}
```

Comment:
why didn't used `{{if this.model.event.isSponsorsEnabled}}` directly here inside if, why to make a computed function?

Reply:
I haven't noticed that. Yup we can do that